<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Layers</class>
 <widget class="QWidget" name="Layers">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>327</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="bottomMargin">
    <number>9</number>
   </property>
   <item row="1" column="0">
    <widget class="QTreeWidget" name="layertree">
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <property name="columnCount">
      <number>0</number>
     </property>
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="closeBtn">
       <property name="text">
        <string>Закрыть</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QToolButton" name="btnActive">
        <property name="minimumSize">
         <size>
          <width>26</width>
          <height>26</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>26</width>
          <height>26</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Сделать слой активным</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/meteo/icons/tools/star.png</normaloff>:/meteo/icons/tools/star.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnUp">
        <property name="minimumSize">
         <size>
          <width>26</width>
          <height>26</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>26</width>
          <height>26</height>
         </size>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Поднять слой на один шаг вверх&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/meteo/icons/tools/layer_up.png</normaloff>:/meteo/icons/tools/layer_up.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnDown">
        <property name="minimumSize">
         <size>
          <width>26</width>
          <height>26</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>26</width>
          <height>26</height>
         </size>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Опустить слой на один шаг вниз&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/meteo/icons/tools/layer_down.png</normaloff>:/meteo/icons/tools/layer_down.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnRemove">
        <property name="minimumSize">
         <size>
          <width>26</width>
          <height>26</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>26</width>
          <height>26</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Удалить слой</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/meteo/icons/tools/delete.png</normaloff>:/meteo/icons/tools/delete.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
